practices/src/test/java/com/kingland/practice/buffer/LinkedBlockingQueueBufferTest.java

```diff
@@ -0,0 +1,88 @@
+package com.kingland.practice.buffer;
+
+import org.junit.jupiter.api.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class LinkedBlockingQueueBufferTest {
```

Comment:
PR2: I don't think the test coverage is enoug, you may need to add more test cases.